device/arista/x86_64-arista_7050cx3_32s/Arista-7050CX3-32S-D48C8/td3-a7050cx3-32s-48x50G+8x100G.config.bcm

```diff
@@ -1,3 +1,13 @@
+host_as_route_disable=1
```

Comment:
For all TD3 config.bcm files will also require a line similar to the one here:

sai_load_hw_config=/etc/bcm56870_a0/b870.6.6.1/

I will disclose the specific path once we have validate this change.

Reply:
@tahmed-dev SAI4.3 PR merged today.  This PR should be able to go as well.
Please make sure you have committed the additional soc property on all TD3 config.bcm file with the following:
sai_load_hw_config=/etc/bcm/flex/bcm56870_a0/b870.6.6.1/
The last "/" is important. missing it will cause syncd on TD3 to crash...